gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/model/ApiKeyMongo.java

```diff
@@ -31,6 +33,8 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@ToString(of = { "key", "expireAt", "revoked", "revokedAt" })
```

Comment:
From what I see in the Lombok docs there is a new way of defining included fields. The one you use is old-fashioned and may be deprecated in time. How about switching to the new one? Wdyt? 

https://projectlombok.org/features/ToString  
<img width="764" alt="image" src="https://github.com/user-attachments/assets/030c6e3b-b9b1-4fa5-8cd5-58f115943aaa">